contracts/Initializable.sol

```diff
@@ -6,6 +6,12 @@ contract Initializable {
 
     event InitializationComplete();
 
+    modifier onlyInitializationAdmin() {
```

Comment:
use the modifier to initializationComplete() instead of 
require(msg.sender == initializationAdmin(), "caller is not the initialization manager");